[ja] Sync docs/concepts/services-networking/ingress-controllers

ystkfujii opened the pull request:
Sync docs/concepts/services-networking/ingress-controllers

Edited Page on Deploy Preview: 
https://deploy-preview-40422--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/services-networking/ingress-controllers/

t-inu reviewed (commented):
I found some updated links compared to en page.
So, I left comments about these links as suggested changes.

ystkfujii commented:
@t-inu 
Thanks for your review.
I updated about your comments. And additionally, I fixed the traefik link. https://github.com/kubernetes/website/pull/40422/commits/11bc549f3cc1d58aaa5f38a209bc3623ece4bf00

Could you please review again ?

t-inu commented on content/ja/docs/concepts/services-networking/ingress-controllers.md:
```diff
@@ -21,24 +24,38 @@ Ingressリソースが動作するためには、クラスターでIngressコン
+* [Pomerium Ingress Controller](https://www.pomerium.com/docs/k8s/ingress.html)は[Pomerium](https://pomerium.com/)ベースのものであり、コンテキストを考慮したアクセスポリシーを提供します。
 * [Skipper](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/)は、カスタムプロキシーを構築するためのライブラリーとして設計された、Kubernetes Ingressなどのユースケースを含む、サービス構成用のHTTPルーターとリバースプロキシーです。
+* [Tyk Operator](https://github.com/TykTechnologies/tyk-operator)はAPI管理機能をIngressに持たせるためにCustom ResourcesでAPIを拡張します。Tyk OperatorはOpen Source Tyk GatewayとTyk Cloudコントロールプレーンで動作します。
 * [Traefik Kubernetes Ingress provider](https://doc.traefik.io/traefik/providers/kubernetes-ingress/)は、[Traefik](https://github.com/containous/traefik) proxy向けのIngressコントローラーです。
```
> https://github.com/containous/traefik -> https://github.com/traefik/

Sorry, I made a mistake.
https://github.com/containous/traefik -> https://traefik.io/traefik/
This is now fixed correctly.

t-inu reviewed (commented):
/lgtm

ystkfujii commented:
/assign @nasa9084

Could you review this?

nasa9084 commented on content/ja/docs/concepts/services-networking/ingress-controllers.md:
```diff
@@ -1,6 +1,9 @@
+description: >-
+  クラスターで[Ingress](/ja/docs/concepts/services-networking/ingress/)を動作させるためには、_ingress controller_ が動作している必要があります。
+  少なくとも1つのIngressコントローラーを選択し、クラスター内にセットアップされていることを確認する必要があります。
+  このページはデプロイ可能な一般的なIngressコントローラーをリスト表示します。
```
```suggestion
  このページはデプロイ可能な一般的なIngressコントローラーをリストアップします。
```

ystkfujii commented:
@nasa9084 

Thanks for your review. 
I updated about your comments. And I also did squash.

ystkfujii commented:
@t-inu 

Would you add the LGTM label ?

t-inu reviewed (commented):
/lgtm

nasa9084 reviewed (commented):
/approve
sorry to be late~
